index.html

```diff
@@ -21,39 +27,29 @@
     <script src="https://js.arcgis.com/3.16/"></script>
     <script src="node_modules/esri-system-js/dist/esriSystem.js"></script>
   </head>
-
+  
   <!-- 3. Display the application -->
-  <body class="calcite">
```

Comment:
removing this class from the body is probably what's causing the issues w/ search styles